src/effects/native/bessel4lvmixeqeffect.cpp

```diff
@@ -10,12 +10,13 @@ QString Bessel4LVMixEQEffect::getId() {
 EffectManifest Bessel4LVMixEQEffect::getManifest() {
     EffectManifest manifest;
     manifest.setId(getId());
-    manifest.setName(QObject::tr("Bessel4 LV-Mix EQ"));
-    manifest.setShortName(QObject::tr("Bessel4 EQ"));
+    manifest.setName(QObject::tr("Bessel4 LV-Mix ISO"));
+    manifest.setShortName(QObject::tr("Bessel4 ISO"));
     manifest.setAuthor("The Mixxx Team");
     manifest.setVersion("1.0");
     manifest.setDescription(QObject::tr(
-        "A Bessel 4th order filter equalizer with Lipshitz and Vanderkooy mix (bit perfect unity, roll-off -24 db/Oct). "
+        "A Bessel 4th order filter isolator with Lipshitz and Vanderkooy mix (bit perfect unity, roll-off -24 db/Oct).") +
```

Comment:
If we're going to pay the cost of retranslating all of these, let's do some more tweaks :). WDYT?

"4th order" -> "4th-order"
"db" -> "dB"
"Oct" -> "octave"

(same for all others effects)

Also, while I find it awkward with the adjectives, I think what you have for the biquad EQ is correct English: "An 3-band EQ...".  So maybe these should all be "An" too since "An" refers to "isolator".